src/components/organisms/HomeContent.tsx

```diff
@@ -109,12 +80,20 @@ export default function HomeContent(): ReactElement {
               }
             >
               <div className={styles.interactivity}>
-                {interactiveComponents[paragraph.interactivity]}
+                <InteractiveModalImage
+                  src={paragraph.image.childImageSharp.original.src}
+                  alt="Compute-to-Data and its benefits"
```

Comment:
issue/suggestion(blocking): we should make sure to always use an alt text associated with the images. I suggest to simpl use the title of the section here for now.